quiz-app/src/assets/translations/hin.json

```diff
@@ -0,0 +1,1017 @@
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "एसपीआई प्रोटोकॉल का समर्थन करता है:",
+						"answerOptions": [
+							{
+								"answerText": "केवल एक नियंत्रक और केवल एक परिधीय",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "केवल एक नियंत्रक और कई परिधीय",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "कई नियंत्रक और कई परिधीय",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "I2C प्रोटोकॉल का समर्थन करता है:",
+						"answerOptions": [
+							{
+								"answerText": "केवल एक नियंत्रक और केवल एक परिधीय",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "केवल एक नियंत्रक और कई परिधीय",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "कई नियंत्रक और कई परिधीय",
+								"isCorrect": "true"
+							}
+						]
+					}
+				]
+			},
+			{
+				"id": 13,
+				"title": "पाठ 7- स्वचालित पौधे पानी: प्री-लेक्चर क्विज",
+				"quiz": [
+					{
+						"questionText": "Iot उपकरणों पानी पंपों को नियंत्रित करने के लिए काफी शक्तिशाली हैं",
+						"answerOptions": [
+							{
+								"answerText": "True",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "False",
+								"isCorrect": "true"
+							}
+						]
+					},
+					{
+						"questionText": "एक्ट्यूएटर का उपयोग अतिरिक्त उपकरणों के लिए शक्ति को नियंत्रित करने के लिए किया जा सकता है",
+						"answerOptions": [
+							{
+								"answerText": "True",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "False",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "सेंसर तुरंत एक्ट्यूएटर से परिवर्तन का पता लगाते हैं",
+						"answerOptions": [
+							{
+								"answerText": "True",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "False",
+								"isCorrect": "true"
+							}
+						]
+					}
+				]
+			},
+			{
+				"id": 14,
+				"title": "पाठ 7-स्वचालित पौधा पानी: पोस्ट लेक्चर प्रश्नोत्तरी",
+				"quiz": [
+					{
+						"questionText": "एक रिले किस प्रकार का स्विच है:",
+						"answerOptions": [
+							{
+								"answerText": "Electrical",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "Electromechanical",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "Mechanical",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "एक रिले की अनुमति देता है:",
+						"answerOptions": [
+							{
+								"answerText": "एक उच्च शक्ति डिवाइस को नियंत्रित करने के लिए एक कम शक्ति डिवाइस",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "कम पावर डिवाइस को नियंत्रित करने के लिए एक उच्च शक्ति उपकरण",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "उनके बीच एक बैटन गुजरते हुए एक दौड़ चलाने के लिए धावक",
```

Comment:
I am not getting the meaning of this option

"उनके बीच एक बैटन गुजरते हुए एक दौड़ चलाने के लिए धावक",